Add DeaDBeeF (#252)
myfreeweb/awesome-gtk: README.md

#### Music Players

- [Amberol](https://apps.gnome.org/Amberol) - Simple music player well integrated with GNOME `#rust` `#gtk4` `#libadwaita` `#gnome`.

> - [DeaDBeeF](https://deadbeef.sourceforge.io/) - Modular cross-platform music player for desktop operating systems `#c` `#gtk3`.

Reviewer: ```suggestion
- [DeaDBeeF](https://deadbeef.sourceforge.io) - Modular cross-platform music player with conversion and tagging features `#c` `#gtk3`.
```
